client/components/TestRun/index.jsx

```diff
@@ -1011,6 +1011,10 @@ const TestRun = () => {
                                               )
                                     }
                                     testPageUrl={testPlanVersion.testPageUrl}
+                                    testFormatVersion={
+                                        testPlanVersion.metadata
+                                            .testFormatVersion
```

Comment:
Big ask here: can the testFormatVersion be included inside the renderableContent?

It's a pretty sizable change, but here's my reasoning about why it might be worth it. 

When contributors come across this variable, called testFormatVersion, the implication is that there can be multiple ways tests are formatted. It's right there in the name. So what's going to happen is that they are going to doubt whether they can rely on the documented concepts like "scenarios" and "assertions" and "tests" and "testResults."

As you know, in reality all these concepts are completely unaffected by the testFormatVersion. But it seems like they could be. That's the disconnect that I want to try to mitigate.

If the testFormatVersion was only accessible from within the renderableContent, A.K.A. renderableContent.testFormatVersion, suddenly it becomes immediately apparent where the safe zone ends and where the danger zone starts. It becomes obvious that if you are attempting to extract some kind of data from the renderableContent, you should make sure to account for all the versions that the data can be formatted.

There reason it is a big ask is that this field is not currently included in the renderableContent and I think it would need to be added in a migration. But furthermore, to do it right, it seems like going forward the field would need to be added into the build process of the ARIA-AT repo for both v1 and v2 build processes.

That's a lot, definitely reach out if I can clarify anything I said above.

Reply:
Answered by https://github.com/w3c/aria-at-app/pull/840#discussion_r1402645001, which is also where that change would happen.